teaching-platform/db_handler.py

```diff
@@ -0,0 +1,100 @@
+import logging
```

Comment:
I suggest moving all stuff related to validating password/ validating username /creating hashes into another module - those are not strictly a part of `db_handler` and this module will be a bit cleaner.

Reply:
Done